src/Fetch.mli

```diff
@@ -212,7 +212,20 @@ module Response : sig
   external text : string Js.Promise.t = "" [@@bs.send.pipe: t]
 end
 
+module FormData : sig 
+  type t = formData
+
+  external make : unit -> t = "FormData" [@@bs.new]
+
+  external mutableAppendObject : string -> < .. > Js.t -> unit = "append" [@@bs.send.pipe : t]
```

Comment:
This method, and `set`, doesn't take an object, but either a `string` or a `Blob`. I don't think an object works as a replacement for a `Blob` either, since it's missing the binary data.

Reply:
oh yeah, not pretty sure if this is the same case for the web, but for React Native you can.
e.g.
```

function getRequestBodyWithUploadables({ operation, variables, uploadables }) {
  const formData = new FormData();
  formData.append('query', operation.text);
  formData.append('variables', JSON.stringify(variables));

  uploadables.forEach((uploadable, key) => {
    formData.append(`image${key}`, {
      name: uploadable.filename || `image${key}.jpg`,
      type: uploadable.mime || 'image/jpg',
      uri: uploadable.path,
    });
  });

  return formData;
}
```

Reply:
I'm going to take a look in the polyfill for it

Reply:
Indeed this, https://github.com/facebook/react-native/blob/master/Libraries/Network/FormData.js#L33
I was basing myself only in the RN version, I'm going to add a `appendBlob` as well then

Reply:
Alright, could you also restrict this to something about a bit more specific than "any object"? It seems to require a fixed shape.

Reply:
And give it a more meaningful name than just `appendObject`

Reply:
oh yeah, good point